selfdrive/controls/controlsd.py

```diff
@@ -191,6 +192,10 @@ def update_events(self, CS):
     if self.sm['deviceState'].memoryUsagePercent > (90 if TICI else 65):
       self.events.add(EventName.lowMemory)
 
+    # Permanent alert for joystick debug mode
+    if self.debug_mode:
```

Comment:
add this as a static alert like here https://github.com/commaai/openpilot/blob/f09c2bd7bde44fd01f3cbf68f5a3f665049d7a67/selfdrive/controls/controlsd.py#L158